pyface/ui/qt4/clipboard.py

```diff
@@ -49,7 +49,7 @@ def _get_object_data(self):
         mime_data = cb.mimeData()
         if mime_data.hasFormat(PYTHON_TYPE):
             serialized_data = BytesIO(mime_data.data(PYTHON_TYPE).data())
-            klass = load(serialized_data)
+            _ = load(serialized_data)
```

Comment:
I don't understand the point of this line given the following line

I assume this line needs to stay though hence the change / not a deletion?

Reply:
it looks like we need to load the data twice - and the first time we call `load`, it returns the `klass` information and the second time we call `load`, we get the actual object from the clipboard. I had originally removed the call entirely - and I saw test failures - which is how I know that we need two calls. See CI failures in the first commit of this PR

Reply:
i added comments in the code reflecting the same